server/controllers/user.js

```diff
@@ -1,4 +1,4 @@
-import bcrypt from "bcrypt";
+import bcrypt from "bcryptjs";
```

Comment:
I think imports will also need a refactor wherever we are importing `bcrypt`.

Reply:
Can you explain more about refractor. I didnt understand your message

Reply:
What I mean is we might need to change the import statements as well where we are using `bcrypt` to maybe `bcryptjs`. Refer this https://github.com/dvjsharma/Drawn2Shoe/blob/a2b35755ef39450325fbda1e87a46bf180e0612f/server/controllers/user.js#L1

Reply:
Oh okay, I already checked all the files. Thats the only one importing bcrypt

Reply:
After making the adjustment, ensure to thoroughly test the functionality end-to-end. The password hashing process should function properly.

Reply:
Only the name is different, all the methods are same in both the modules
Everything is working same as before

<img width="391" alt="Screenshot 2024-06-01 174333" src="https://github.com/dvjsharma/Drawn2Shoe/assets/104003078/a242fd0e-d946-4e36-ada4-66e7dfbc31c6">